src/index.js

```diff
@@ -74,3 +76,13 @@ process.on('SIGINT', async () => {
     process.exit(1);
   }
 });
+
+if (process.env.NOTIFY_METHOD) {
```

Comment:
If we put cron job and web server together, there may be a case that if we have multiple instances for web server, the cron job will be triggered once for each instance, causing the job be run multiple times.

I would suggest we just prepare a standalone NodeJS script that can be invoked via command line.
On other environments, people can use native cron tab to trigger the script; and on Heroku, we can use [Heroku scheduler](https://devcenter.heroku.com/articles/scheduler) to schedule a run.